navigation/AppNavigator.js

```diff
@@ -18,11 +18,11 @@ const App = createStackNavigator({
   Main: MainTabNavigator
 });
 
-// export default createSwitchNavigator({
+export default createSwitchNavigator({
 //   // You could add another route here for authentication.
 //   // Read more at https://reactnavigation.org/docs/en/auth-flow.html
 
-//   Main: MainTabNavigator,
-// });
+  Main: MainTabNavigator,
+});
 
-export default App;
\ No newline at end of file
+//export default App;
```

Comment:
Can we do `export default App` instead?

Reply:
@kmui2  it didn't run with export default app. If I had it uncommented, then Dan's registration screen would appear (which didn't do anything) and I wasn't able to view the profile screen with the nav bar and test out my code

Reply:
i would  see dan's registration screen and nothing would happen. I commented to test out my code